plugins/grafana-custom-plugins/grafana-dependency-plugin/README.md

```diff
@@ -42,18 +42,24 @@ loading of data for the Service Dependency Graph Plugin, the query is expected
 to return the following fields, in arbitrary order.
 
 - field 1: sourcePodName value with name or an alias of `sourcePodName`
-- field 2: sourceNodeName value with name or an alias of `sourceNodeName`
-- field 3: destinationPodName value with name or an alias of `destinationPodName`
-- field 4: destinationNodeName value with name or an alias of `destinationNodeName`
-- field 5: destinationServicePortName value with name or an alias of `destinationServicePortName`
-- field 6: octetDeltaCount value with name or an alias of `octetDeltaCount`
+- field 2: sourcePodLabels value with name or alias of `sourcePodLabels`
+- field 3: sourcePodNamespace value with name or alias of `sourcePodNamespace`
```

Comment:
why we add `sourcePodNamespace` here? Is it being used anywhere in the source code?